test-pup/example-react-app/form-controls/text-field.test.js

```diff
@@ -24,6 +24,7 @@ describe('TextField form control', () => {
     const $document = await getDocument(page);
 
     const $field = await findByLabelText($document, 'Manufacturer');
+    await $field.click({clickCount: 3});
```

Comment:
It will be useful to add a comment that this is a technique to select existing text so that it gets overwritten by subsequent `type`. Otherwise it's not obvious.